### ISSUE TITLE
Export: dependencies are not detected when using load() to load resources

### ISSUE THREAD

**Issue by avnerh1** [godotengine/godot#47586]
3.2.3.stable.mono
Win10

**Issue description:**
When exporting, and choosing "Export selected resources" or "Export selected scenes" under "Resources" tab, dependent resources are auto-detected and exported - but not if dependency is by a `load()` directive.
This makes the feature much less reliable and not turnkey. It is advisable that the exported scripts will be parsed and resources that are literally mentioned in them using `load()` will be automatically export too.

**Steps to reproduce:**
1. Project->Export...
2. Resources tab
3. Export Mode: "Export selected resources" or "Export selected scenes"
4. Under "Resources to export:", select a script (or a scene that has uses a script) that loads a resource using `load("res://...") `
5. Click "Export Project"
6. Run the exported project on the target platform, and see that this resource is missing

**Comment by volzhs** [comment]
it won't be possible to detect while exporting.
because it can be any path depending on state.
```
load(stage_path + index)
load(stage_path + difficulty)
load(stage_path + choice)
```

**Comment by smix8** [comment]
I don't think this is possible cause load() paths can be to dynamic.

Also as long as there is no valid option to prevent the editor from exporting redundant dependencies in every single package file I don't want such automated help. I had to remove every use of preload() in my game due to this and depend on load() without forced dependencies export way to much to keep my project export size small. If something like this ever gets added make it a toggle option so I can turn it off please.

**Comment by avnerh1** [comment]
Of course, any fix would address only literal resource paths (or a very basic parsing of, say, two const strings concatenated) but not dynamic ones.
I'm not sure how many people use dynamic resource paths. I personally don't, and if many people are like me, fixing this issue would be beneficial.
